### PR TITLE
Raspi installation page improved

### PR DESCRIPTION
**Pull request by b286**

### CODE REVIEW

File: GUI/RASPBERRY_PI/README.md [hunk 1]
```diff
@@ -112,166 +193,229 @@ For downloading the files, first create two new directories:
...
-    $ which conda 
+    $ which conda
     $ which python
     $ source activate UC2env
     $ which python
     $ python
     $ exit()
     ```
-    which will should look something like this: 
+    which will should give you this as a result:
```
Comment (renelachmann) [inline GUI/RASPBERRY_PI/README.md]: erase a "will"

File: GUI/RASPBERRY_PI/README.md [hunk 2]
```diff
@@ -112,166 +193,229 @@ For downloading the files, first create two new directories:
...
     ```
     $ python main.py
     ```
-    It should start without any errors, but maybe it is overscaled on the first start. Just restart it and it should work. Press "CUS" on the left to get the LED-array-manipulation window and press "Start Preview" on the top to start your camera-stream. 
+    It should start without any errors, but maybe it is overscaled on the first start. Just restart it and it should work.
+* Quick test: Press "Start Preview" (using the touchscreen or the mousepad) on the top to start your camera stream. Press "CUS" on the left to get the LED-array-manipulation window and by pressing the blocks that represent the LEDs, create a smiley face.
     <p align="center">
     <img src="./IMAGES/script10.png" width="400" alt="">
     </p>   
+* **Congratulations! You made it! Your RasPi is now smiling at you :-)**
```
Comment (renelachmann) [inline GUI/RASPBERRY_PI/README.md]: 👍

**Review by renelachmann — COMMENTED** [review]
- GUI/RASPBERRY_PI/IMAGES/pi-desktop.png --> will change depending on RasPi-OS used etc -> I like it there, but please add a hint that this could look different on different versions...
- GUI/RASPBERRY_PI/IMAGES/script11.png --> please add green rectangle around important line
- regarding all images in the repo --> to reduce traffic would it be possible to exchange **ALL** non-vectorial images with web-optimized (~60-70% quality-rate) JPG-images ? -> will speed-up page and reduce the whole archive

Thank you for the efforts!!!

**Comment by b286** [comment]
Thank you for the comments, it's implemented now. 

The images can surely becom JPGs, but that I'll do later.

**Pull request merged**